cli_tools/gce_vm_image_import/importer/api_inflater_test.go

```diff
@@ -84,3 +91,62 @@ func TestCreateAPIInflater_IncludesUEFIGuestOSFeature(t *testing.T) {
 	assert.Contains(t, realInflater.guestOsFeatures,
 		&computeBeta.GuestOsFeature{Type: "UEFI_COMPATIBLE"})
 }
+
+func TestAPIInflater_Inflate_CreateDiskFailed_CancelWithoutDeleteDisk(t *testing.T) {
```

Comment:
In addition to unit tests, can you add some e2e tests that fail  prior to your fix, but pass once it's applied?

Reply:
added and tested before/after